server/controllers/api/videos/index.ts

```diff
@@ -112,18 +119,27 @@ videosRouter.get('/',
   commonVideosFiltersValidator,
   asyncMiddleware(listVideos)
 )
+
+videosRouter.post('/upload',
+  authenticate,
+  reqVideoFileAdd,
+  asyncMiddleware(videosAddValidator),
+  asyncRetryTransactionMiddleware(addVideoLegacy)
+)
+
+videosRouter.use('/upload-resumable',
```

Comment:
`/resumable-upload`?

Reply:
I meant `-resumable` as a suffix to convey the notion this is a variant.